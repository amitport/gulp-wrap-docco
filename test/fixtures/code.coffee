# testing some *code*

fn = ->
	for x in arr # this is a loop
		console.log x

# this is the end of the code