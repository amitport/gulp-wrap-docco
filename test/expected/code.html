
  
  <p>testing some <em>code</em></p>

  
    <div class='highlight'><pre>
<span class="hljs-function"><span class="hljs-title">fn</span> = -&gt;</span>
  <span class="hljs-keyword">for</span> x <span class="hljs-keyword">in</span> arr <span class="hljs-comment"># this is a loop</span>
    <span class="hljs-built_in">console</span>.log x</pre></div>
  

  
  <p>this is the end of the code</p>

  

